samples/DerivedHandlers/README.md

```diff
@@ -0,0 +1,9 @@
+AuthSamples.DerivedHandlers
```

Comment:
I feel this doc requires more details. For example, you mention `// Normally Identity handles sign in, but you can do it directly` in `AccountController`. Like mentioning the differences in what by default happens vs. what this sample does probably would be helpful.

Reply:
Do you mean just expanding on the comments?  The cookie samples aren't really focusing on the identity generation part of things, but I guess its probably worth at least pointing to a real doc/topic talking about this